tests/end-to-end/backup/pkg/tests/eventmesh/application.go

```diff
@@ -0,0 +1,189 @@
+/*
+Copyright 2020 The Kyma Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eventmesh
+
+import (
+	"fmt"
+
+	"github.com/avast/retry-go"
+
+	scv1beta1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	serviceCatalogCS "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
```

Comment:
It's a good practice to avoid capital letters in imports to avoid confusion with variables names. A nice reference: https://rakyll.org/style-packages/#lowercase-only

Reply:
done